templates/institutional-landing/institutional-landing.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @title Institutional landing page template detection
+ * @overview Logic to detect pages that seem to be institutional landing pages without the appropriate class
```

Comment:
Can you mention this is temporary measure until that proper fix is implemented by content management system. This code are going to be deprecated and removed at the same time of the version 1 of the ILP template. It just need to be aligned with our technical implementation note.